man/set-content-if-missing.md

```diff
@@ -0,0 +1,29 @@
+## set content if missing
+
+Creates a function that when passed to d3's `selection.select` operator will conditionally set the HTML content of a node (only if it doesn't contain something that matches the provided selector) and makes a selection using the selector.
```

Comment:
Like above, if the template must contain only a single root node, we can rephrase this to be a bit more descriptive I think:

> Creates a function that when passed to d3's `selection.select` operator will set the contents of a node *only if* it *does not* match the provided selector, and selects the root node of the supplied template.